recipes/inaSpeechSegmenter/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "inaSpeechSegmenter" %}
+{% set version = "0.7.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/inaSpeechSegmenter-{{ version }}.tar.gz
+  sha256: 506b193ae39a1a19b1e9219cf58493208152ca43f90a41122336ef9612efde9d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }},<3.13
+    - ffmpeg 
+    - tensorflow   #not avalaible yet on py313 and avalaible windows versions are for older unsupported python version
+    - numpy
+    - pandas
+    - scikit-image
+    - pyannote.core
+    - matplotlib-base
+    - pyro4
+    - pytextgrid
+    - pysoundfile
+    - onnxruntime *=*_cuda   #not avalaible for osx
+    - __linux
```

Comment:
@conda-forge/help-python I'm not sure if this is the right way for a noarch package only avalaible for linux

Reply:
@ocefpaf what are the indications?

Reply:
Things like:

```
<<<<<<< HEAD
```

but I guess I was looking at an old diff.

Reply:
So is the recipe correct?